torchtune/utils/__init__.py

```diff
@@ -22,6 +22,7 @@
     validate_no_params_on_meta_device,
     wrap_fsdp,
 )
+from ._generation import generate  # noqa
```

Comment:
nit: why do you need the #noqa?

Reply:
Pls someone add some analogous version of [this](https://github.com/pytorch/vision/blob/2c4665ffbb64f03f5d18016d3398af4ac4da5f03/setup.cfg#L14-L15) to our .flake8, then we can delete all these godforsaken noqas in our init files

Reply:
I know you've been asking for this @ebsmothers! I'm going to be a pain and punt this to a follow up!